database/schema/12-feegrant.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE fee_grant_allowance
+(
+    id                 SERIAL      NOT NULL PRIMARY KEY,
+    grantee            TEXT        NOT NULL REFERENCES account (address),
+    granter            TEXT        NOT NULL REFERENCES account (address),
+    allowance          JSONB       NOT NULL,
+    height             BIGINT      NOT NULL
```

Comment:
The `feegrant` module does not allow having more than one allowance per grantee-granter at the same time. For this reason, we should add a `UNIQUE` constraint here: 
```suggestion
    height             BIGINT      NOT NULL,
    CONSTRAINT unique_fee_grant_allowance UNIQUE(grantee, granter) 
```